### PR TITLE
feat BackendTrafficPolicy

### PR DESCRIPTION
**Pull request by arkodg**
Takes PR https://github.com/envoyproxy/gateway/pull/1961 forward

### CODE REVIEW

**Review by AliceProxy — DISMISSED** [review]
Thanks for taking this forward :pray:

**Review by AliceProxy — DISMISSED** [review]

File: site/content/en/latest/design/backend-traffic-policy.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
+---
+title: "BackendTrafficPolicy "
```
Comment (zirain) [inline site/content/en/latest/design/backend-traffic-policy.md:2]:
```suggestion
title: "BackendTrafficPolicy"
```

File: site/content/en/latest/design/backend-traffic-policy.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+- If Policy A has a `targetRef` that includes a `sectionName` i.e.
+it targets a specific Listener within a `Gateway` and Policy B has a `targetRef` that targets the same
+entire Gateway then
+  - Policy A will be applied/attached to the specific Listener defined in the `targetRef.SectionName`
+  - Policy B will be applied to the remaining Listeners within the Gateway. Policy B will have an additional
+  status condition `Overridden=True`.
+
+## Alternatives
+
+- The project can indefintely wait for these configuration parameters to be part of the [Gateway API].
```
Comment (zirain) [inline site/content/en/latest/design/backend-traffic-policy.md:153]:
```suggestion
- The project can indefintely wait for these configuration parameters to be part of the [Gateway API][].
```

File: site/content/en/latest/design/backend-traffic-policy.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+- Load balancing
+
+## Design Decisions
+
+- This API will only support a single `targetRef` and can bind to only a `Gateway` or xRoute (`HTTPRoute`/`GRPCRoute`/etc.) resource.
+- This API resource MUST be part of same namespace as the resource it targets.
+- There can be only be ONE policy resource attached to a specific `Listener` (section)  within a `Gateway`
+- If the policy targets a resource but cannot attach to it, this information should be reflected
+in the Policy Status field using the `Conflicted=True` condition.
+- If multiple polices target the same resource, the oldest resource (based on creation timestamp) will
```
Comment (zirain) [inline site/content/en/latest/design/backend-traffic-policy.md:142]: just want to make sure all EG's CRD follow same rule.

**Review by zirain — DISMISSED** [review]
LGTM with small nits.

File: site/content/en/latest/design/backend-traffic-policy.md (hunk 3 above)
Comment (arkodg) [reply site/content/en/latest/design/backend-traffic-policy.md:142]: yes all policies follow similar rule

**Review by zirain — APPROVED** [review]

**Pull request merged**